package.json

```diff
@@ -59,6 +60,7 @@
     "@web-stories-wp/tracking": "*",
     "@web-stories-wp/units": "*",
     "@web-stories-wp/widget": "*",
+    "@web-stories-wp/wp-dashboard": "*",
     "@web-stories-wp/wp-story-editor": "*",
     "@wordpress/api-fetch": "^5.2.1",
     "flagged": "^2.0.1",
```

Comment:
We can now cleanup unused top level dependencies here, e.g. remove `query-string` and `history` since they're declared in `dashboard/package.json`. Possibly more.

Reply:
Yes, except `react` and `styled-components` because they are being used in .storybook.
Done in https://github.com/google/web-stories-wp/pull/8646/commits/db4a8ab85acb786781c94f8bccab3da8fbdff132